cypress/integration/osd-bundle/check_loaded_data.js

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
This should have the shortened license header.

Reply:
will fix